### DISCUSSION TITLE
Kickstart Open Source with Flutter

### DISCUSSION THREAD

**Discussion by sandeshkolte** [community/community#110703] [GitHub Education]
### Hi everyone,

I would like to know if there's an easy to implement way to contribute to an Open Source as a Flutter and Nodejs User.

**Comment by SoroushBeigi** [comment]
Hi. there are many ways to contribute to open source and I will point out to some:
Flutter Issues: Search for projects on GitHub tagged with "Flutter" and filter issues for "beginner", "help wanted" and "good first issue" labels. This helps you find projects looking for contributions that match your skillset.
Explore the Ecosystem: Look beyond official Flutter repositories. Search for interesting open-source projects on GitHub that use Flutter and see if they have contribution guidelines.
You don't have to start with big contributions, updating old sdk versions, fixing null safety problems and even improving the documentation would help a lot. just fork a repository, commit your changes and do a PR.
Make sure to review the repository's guidelines, big repositories like flutter itself have some guidelines you have to follow.

**Comment by Musaddiq625** [comment]
@sandeshkolte 
Every open source community has some guidelines and some rules
if you want to contribute in the Flutter, make sure to follow these links:
[Flutter](https://github.com/flutter/flutter?tab=readme-ov-file#documentation)
[Node JS](https://github.com/flutter/flutter?tab=readme-ov-file#documentation)
